### DISCUSSION TITLE
Ant Design Pro with NextJs 14

### DISCUSSION THREAD

**Discussion by xXAvoraXx** [ant-design/ant-design-pro#11299] [Show and tell]
Hello everyone, I love using UmiJs, but the lack of UmiJs documentation makes it difficult for us European developers to adopt it, or rather the awareness of UmiJs is low. So I propose to develop a NextJs version of an advanced system like Ant Design Pro together with the community.

https://github.com/xXAvoraXx/nextjs-antd-pro

**Comment by lanmingle** [comment]
很棒！
不过我建议不要采用 umijs ，这个维护能力让人担忧，我们还是选择 nextjs 吧！

Nice!
But I recommend not using umijs, this maintenance capability is worrying, we still choose nextjs!

---

抱歉，我是 react 和 nextjs 新人，参与不了这项工作，希望你能成立个组织，把这个项目坚持和传承下去。

Sorry, I am new to react and nextjs and can't participate in this work. I hope you can form an organization to keep this project going and pass it on.

**Reply by xXAvoraXx** [reply]
Ant Design 的 nextjs 版本可在 refine.dev 解决方案中使用，只是没有专业插件。
The nextjs version of Ant Design is available in refine.dev solutions, only there are no pro plugins.
https://refine.dev/
